shared/src/common/enums/validation/sign-in-validation-message.enam.ts

```diff
@@ -0,0 +1,12 @@
+import { SignInValidationRule } from './sign-in-validation-rule.enam';
```

Comment:
Fix all enum names
`shared/src/common/enums/validation/sign-in-validation-message.enam.ts` => shared/src/common/enums/validation/sign-in-validation-message.enum.ts